docs/configuration/http_filters/router_filter.rst

```diff
@@ -219,6 +219,8 @@ prefix <config_http_conn_man_stat_prefix>` comes from the owning HTTP connection
   no_cluster, Counter, Total requests in which the target cluster did not exist and resulted in a 404
   rq_redirect, Counter, Total requests that resulted in a redirect response
   rq_total, Counter, Total routed requests
+  flow_control_paused_downstream_reads_total, Counter, Total times requests backed up enough to pause reading from downstream.
```

Comment:
IMO for now it's probably overkill, but up to you. But we definitely need to track the cluster that we paused. Are we doing that with this change? I didn't see it.